07-building-chat-applications/typescript/chat-completions-app/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "chat-completions-app",
+  "version": "1.0.0",
+  "description": "a simple application how to use the chat completions with Azure OpenAI in TypeScript",
+  "main": "main.js",
+  "scripts": {
+    "build": "rimraf build && tsc",
+    "preserve": "npm run build",
+    "start": "npm run build && cross-env NODE_ENV=development concurrently \"nodemon\"",
```

the start command can be simplified to this (I don't this need cross-env and concurrently packages):
```suggestion
    "start": "nodemon",
```